jenkins/jobs/macros-common.yaml

```diff
@@ -60,6 +60,16 @@
           #echo "Detailed logs: http://logs.openstack.xenproject.org/$LOG_PATH/"
           echo "Detailed logs: http://dd6b71949550285df7dc-dda4e480e005aaa13ec303551d2d8155.r49.cf1.rackcdn.com/$LOG_PATH/"
 
+- builder:
+    name: copy-dom0-logs
+    builders:
+      - shell: |
+          #!/bin/sh
+          # make dir on jekins server to save Dom0 logs
+          sudo -u domzero mkdir -p ~domzero/logs
+          sudo -u domzero scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null root@{dom0_ip}:/var/log/messages /home/domzero/logs/
```

Comment:
As we talked, messages is not the only logs need to be copied. Please refer to the log list in the ticket. And need take account of the rotated logs e.g. messages.1